tutorials/kube-mqtt/refresher-container/src/run_bridge.py

```diff
@@ -0,0 +1,78 @@
+"""Binary for running Mosquitto MQTT bridge with credential rotation."""
```

Comment:
You should probably add an Apache 2.0 license to the headers.

Reply:
In general, this hasn't been the case for community solutions, somewhat outside OSPO std flow

Reply:
SGTM

Reply:
We have a blanket license statement for the whole community repository, so this is not strictly necessary.